### ISSUE TITLE
Authentic 19.00 don't show caption in tabs

### ISSUE THREAD

**Issue by sz00gun** [qooob/authentic-theme#853]
Theme: Authentic Theme 19.00-git-201710041248
Ubuntu Linux 16.04.2

The bug: please have a look a screen shot: 
![bug](https://user-images.githubusercontent.com/7549566/31338631-fceb9e72-acf7-11e7-8edc-0ac0c4aecefc.jpg)

PS. how to remove those errors?

**Comment by qooob** [comment]
Clear cache and reload the theme.

**Comment by sz00gun** [comment]
Off course I have done it. I purged a cache from the browser, also have a fresh theme installation, I tried beta and stable version. Versions previous than 19 works fine.

**Comment by qooob** [comment]
What is your browser?

**Comment by sz00gun** [comment]
This is not working in Firefox and Chrome

**Comment by qooob** [comment]
Get the latest _git_ version to test it. It's fixed.

**Comment by sz00gun** [comment]
Unfortunately a bug still there... ;(

**Comment by qooob** [comment]
Yes, I know. Just also noticed it.

**Comment by qooob** [comment]
Will try fixing it upon next commit.

**Comment by qooob** [comment]
Go to theme settings, pull latest commit, clear browser's cache forcefully and try again.

It works on my server that also gave an error, should also work on yours.

**Comment by sz00gun** [comment]
What is the latest patch? My server recognises: Authentic Theme 19.00-git-201710101341 only

PS. just different quick question:

could you consider to change the code by pressing RBM on the Webmin or Virtualmin on left top to show the Open Link in new tab.   I just get used to.  Or just by pressing RBM on it to open a new cart with Virtualmin - it was before, not in 19v.

**Comment by qooob** [comment]
Just force update using settings. It's not pushed to updates yet.

I'm not sure what you were asking, could you rephrase the question please.

**Comment by qooob** [comment]
Just force update using settings. It's not pushed to updates yet.

I'm not sure what you were asking, could you rephrase the question please.

**Comment by sz00gun** [comment]
1. I force re installed it by: 
![install](https://user-images.githubusercontent.com/7549566/31411432-8a917a5a-ae09-11e7-8e80-771daa504647.jpg)

2. How to quick open a new tab with virtualmin? 
![wirt](https://user-images.githubusercontent.com/7549566/31411446-9a879e44-ae09-11e7-8e3e-0cc2bc0b01b4.jpg)

**Comment by qooob** [comment]
At first, you never said anything about captions on the right slider?

I will add the ability to open new window from the switch later.

**Comment by sz00gun** [comment]
1. Yes the original of the bug was: captions on the right slider,

2. That open in new window is another minor issue, I just didn't want to open a new ticket.

**Comment by sz00gun** [comment]
Works now... better than before, however still not in 100%. Please have a look a screen cast: 
https://screencast-o-matic.com/watch/cb6X2JIEuS

**Comment by qooob** [comment]
It's obviously in theme's settings. You can enable/disable any tab you want. Check for `Side slider options`.

**Comment by sz00gun** [comment]
Settings are fine, all tabs should be visible all the time, please again, have a look: https://screencast-o-matic.com/watch/cb6XqAIEdw

**Comment by qooob** [comment]
Interesting. What browser is that?

**Comment by sz00gun** [comment]
Latest Firefox, however latest chrome works the same like the Fox

**Comment by qooob** [comment]
Confirmed, thanks a lot. :wink:

**Comment by qooob** [comment]
Fixed in the commit above.

**Comment by qooob** [comment]
Also added opening current page into new tab:
https://github.com/qooob/authentic-theme/issues/853#issuecomment-335612375

You would have to allow popup for your domain.

**Comment by sz00gun** [comment]
allow popup for the domain works, but not as before, and not as it should be, because a new window don't contain a right slider, and left menu. ;(

**Comment by qooob** [comment]
It will contain it, after Jamie adds my patch to the next Webmin release.

You can do it manually:
https://github.com/webmin/webmin/pull/652
